CHANGELOG.rst

```diff
@@ -1,5 +1,41 @@
 .. _changelog:
 
+Renku ``0.25.4`` introduces several UI big-fixes.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🐞 Bug Fixes**
+
+- **UI**: Prevent flashing a spinning wheel when loading workflows
+  (`#2493 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2493>`_).
+- **UI**: Restore the flag to show inactive workflows
+  (`#2502 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2502>`_).
+- **UI**: Prevent infinite spinning wheel when accessing projects by numeric ID
+  without the required user's permissions
+  (`#2476 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2476>`_).
+- **UI**: Open the target notebook when accessing a session from the file preview
```

Comment:
what was fixed here? If I'm interpreting this correctly, can we say:

```suggestion
- **UI**: Fix an issue where a notebook would not automatically open in a session for anonymous users
```

Reply:
Ops I used a very cryptic message. Thanks for spotting it. The suggestion is on point :ok_hand: